CONTRIBUTING.md

```diff
@@ -107,6 +107,26 @@ Here's how to make a one-off code change. (If you're working on a larger feature
 
 7. **Celebrate.** Congratulations, you have contributed to Oppia!
 
+### Notes
+
+Since linting has been added, author tags are omitted in all files. Author tags also tend to be inaccurate or become stale when the author is no longer a regular contributor. 
```

Comment:
We do not use author tags in files, since they tend to be inaccurate or become stale when the author is no longer a regular contributor. However, you can still find the author of a particular change in a file by running the command: